explorer/e2e/anvil/anvil-tabs.ts

```diff
@@ -30,6 +30,28 @@ export const ORGANISM_TYPE_INDEX = 8;
 export const PHENOTYPIC_SEX_INDEX = 9;
 export const REPORTED_ETHNICITY_INDEX = 10;
 
+const anvilDatasetsPreselectedColumns = [
```

Comment:
Right now some of the tests do depend on columns being listed in order. I can change this, but it would involve making changes to tests other than the backpages, so I think it makes more sense to do it in #4078

Reply:
Right now some tests (e.g. `anvil-tabs.spec.ts`) depend on the columns in each array being in order, which is why I kept it as an array other than an object. I agree this is messy, so I can rewrite those not to care about order, although I think it makes sense to include as part of #4078 because it would require wider changes